clients/tabby-agent/src/postprocess/golden/remove_duplication/duplicated_line_suffix.toml

```
@@ -0,0 +1,23 @@
+description = 'Remove duplication: dupicated line suffix'
+
+[config]
+# use default config
+
+[context]
+filepath = 'checks.ts'
+language = 'typescript'
+# indentation = '  ' # not specified
+text = '''
+const log = ({ ok }: { ok:├ boolean }) => { ┤ }) => {
+  console.log(ok);
+}
+'''
+
+[expected]
+# text = '''
+# const log = ({ ok }: { ok:├ boolean┤ }) => {
+#   console.log(ok);
+# }
+# '''
+unchanged = true
```

```suggestion
text = '''
const log = ({ ok }: { ok:├ boolean┤ }) => {
  console.log(ok);
}
'''
```

It is ok to add what is expected here, no need to comment it out. The `notEqual` flag will assert that the output is not equal to the expected result.